README.md

```diff
@@ -1,55 +1,171 @@
+#  #PayTax App
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/21fe87768d424c2e85b089afea2f2084)](https://app.codacy.com/gh/BuildForSDG/PayTax-app?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/PayTax-app&utm_campaign=Badge_Grade_Settings)
 
 ## About
 
-What is this project about. Ok to enrich here or the section above it with an image. 
+This Application aims to enhance the process of tax collection with a platform that allows for a seamless tax payment.
 
-Once this repo has been setup on Codacy by the TTL, replace the above badge with the actual one from the Codacy dashboard, and add the code coverage badge as well. This is mandatory
 
-This is a simple JavaScript starter repo template for setting up your project. The setup contains
-
-- Jest: For runnung tests. We strongly recommend that JavaScript projects use Jest
-
-- Eslint & Prettier: For formatting code to match ESlint AirBnB coding standard. You might need to install the ESlint and Prettier plugins for your code editor to make the most of these utilities
 
 ## Why
 
-Talk about what problem this solves, what SDG(s) and SGD targets it addresses and why these are imoirtant
+-  Tax payers can easily pay tax from the comfort of the their business locations and offices.
+-  People don't have to queue up at offices to pay tax.
+-  The government can save operational cost of sending tax agents to business locations and offices. 
+-  We're essentially functioning as a third party service, we can deduct a token for the service charge.
+-  Easily get tax receipts. 
 
 ## Usage
- How would someone use what you have built, include URLs to the deployed app, service e.t.c when you have it setup
+### API guide for consuming this back-end layer. 
+
+**Individual businesses and Companies registration.**
+Upon registration the Tax payer gets returned a Tax Payer ID by email and json response.
+
+`POST`  https://paytax-app.herokuapp.com/api/v1/users/register
+
+Request format: Individual business
+
+`{
+	"company":"false",
+	"name":"dougals",
+	"address":"lagos",
+	"email":"rickpeterson@gmail.com",
+	"phone":"09060016252",
+	"bvn":"124214141235",
+	"password":"wqdsafdsfds242153",
+	"state":"lagos",
+	"city":"laten town"
+}` 
+
+Request format: Company
+
+` {
+	"company":"true",
+	"name":"peterson",
+	"address":"lagos",
+	"email":"y@y.com",
+	"phone":"09067026252",
+	"bvn":"124214141235",
+	"password":"wqdsafdsfds242153",
+	"state":"lagos",
+	"city":"ogba"
+}`
+
+****
+
+**Individual businesses and Companies login.**
+
+This request returns an access token upon  Authentication or error messages. 
+
+`POST`  https://paytax-app.herokuapp.com/api/v1/user/login
+
+Request format: 
+
+`{
+	"taxID":"SyNcnrmKJy",
+	"password":"password123456"
+}`
+
+****
+
+**Business data for Authenticated users.**
+
+Returns all company and individual biodata.
+
+`GET`  https://paytax-app.herokuapp.com/api/v1/use/biodata/:taxID 
```

Comment:
is this accurate: _api/v1/**use**/biodata/:taxID_

Reply:
:( its a typo. it's api/v1/user/